### PR TITLE
chore: change plugin image to svg

### PR DESCRIPTION
**Pull request by foolwc**
Changes:
copied `iconfont.js` from project `apisix-dashboard` and load plugin icon SVG from it.

### CODE REVIEW

File: website/static/data/plugin.json [hunk 1]
```diff
@@ -184,56 +153,47 @@
...
                 "description": "The ext-plugin-post-req runs after executing builtin Lua plugins and before proxying to the upstream",
-                "src": "/img/plugin/defaultPluginImg.jpg"
+                "useDefaultIcon": true
             }
         ]
     }
-]
-
-
+]
```
Comment (liuxiran) [inline website/static/data/plugin.json]:
```suggestion
]

```

**Comment by liuxiran** [comment]
thanks for your nice work, @foolwc 
also cc @KishaniKandasamy @1502shivam-singh @Baoyuantop to have look

**Comment by 1502shivam-singh** [comment]
@foolwc, Are you going to center the icons in this PR only, given that this closes #484 ?

**Comment by foolwc** [comment]
> @foolwc, Are you going to center the icons in this PR only, given that this closes #484 ?

I will fix that in another PR and #484 maybe needs more discussion for design.  Thanks

**Review by juzhiyuan — APPROVED** [review]

File: website/src/pages/plugins.js [hunk 2]
```diff
@@ -124,20 +124,31 @@ const SectionTitle = styled.h2`
 
 
 function Plugins(props) {
+  useEffect(() => {
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = '/js/plugin-icon.js';
+    document.head.appendChild(script);
+  }, []);
```
Comment (Baoyuantop) [inline website/src/pages/plugins.js:133]: Is it possible to use this method?

https://docusaurus.io/docs/next/docusaurus.config.js/#scripts

**Review by iamayushdas — APPROVED** [review]

File: website/src/pages/plugins.js (hunk 2 above)
Comment (foolwc) [reply website/src/pages/plugins.js:133]: Yes, it works. But it will load this JS globally and maybe reduce the loading speed of other page.

**Review by liuxiran — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by KishaniKandasamy — APPROVED** [review]

**Pull request merged**